src/pages/search/SearchPage.tsx

```diff
@@ -37,17 +37,20 @@ const SearchPage = () => {
             sx={{
               display: 'grid',
               gridTemplateRows: 'auto auto 1fr',
-              gridTemplateColumns: { xs: '1fr', md: '2fr 5fr 2fr' },
+              gridTemplateColumns: { xs: '1fr', md: '2fr auto 5fr 2fr' },
               gridTemplateAreas: {
                 xs: "'searchbar' 'sorting' 'filters' 'advanced' 'results'",
-                md: "'filters searchbar sorting' 'filters advanced advanced' 'filters results results'",
+                md: "'filters divider searchbar sorting' 'filters divider advanced advanced' 'filters divider results results'",
```

Comment:
Skjer noe funky på mobilskjermer nå. Mener å ha vært borti det samme før selv når jeg har brukt grid med conditions hvor enkelte elementer bare skal vises for enkelte skjermstørrelser (som Divider her)
![bilde](https://user-images.githubusercontent.com/6313468/203259738-ac4466f1-0ae6-47aa-907b-213e2e2a9ddb.png)

Reply:
Du kan se i `Header.tsx` hvordan det er gjort for `new-result` som har litt samme case. Tror du må legge til `display: none` på Divider for små skjermer